packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -114,6 +114,31 @@ class MyStack extends Stack {
 
 Note that overriding the `get availabilityZones()` method will override the default behavior for all constructs defined within the Stack.
 
+### Reserving availability zones
+
+There are situations where the IP space for availability zones will
+need to be reserved. This is useful in situations where availability
+zones would need to be added after the vpc is originally deployed,
+without causing IP renumbering for availability zones subnets. The IP
+space for reserving n availability zones can be done by setting the
```

Comment:
I think that n is intentional here to mean that you're not sure how many, but the wording may not sound clear to others.

Reply:
Editing to add that now that I've read the line below, I see that it was indeed intentional, but I think the wording is a little clunky here. Maybe at least make n => `n`.